packages/kotti-ui/source/kotti-popover/components/IconTextItem.vue

```diff
@@ -44,11 +36,26 @@ export default defineComponent<{
 	display: flex;
 	align-items: center;
 	padding: var(--unit-2) var(--unit-4);
-	cursor: pointer;
+	user-select: none;
 	border-radius: var(--border-radius);
 
+	&--is-clickable {
+		cursor: pointer;
+
+		&:not(.kt-popover-options-item--is-disabled) {
+			&:hover {
+				background-color: var(--ui-01);
+				&.kt-popover-options-item--is-selected {
+					color: var(--interactive-01-hover);
+				}
+			}
+		}
+	}
+
 	&--is-disabled {
-		cursor: not-allowed;
+		// `--is-disabled` has same specificity as `--is-clickable`.
```

Comment:
This can actually be fixed by moving `cursor:pointer` into the `:not` below it I think